### ISSUE TITLE
GPU power at 0W

### ISSUE THREAD

**Issue by VoltaVX** [RainbowMiner/RainbowMiner#1515]
Good morning,
I've noticed that RainbowMiner always seems to show GPU as 0W regardless of version. I use a GTX 780, so I'm wondering if having an old GPU is what's causing this.
![image](https://user-images.githubusercontent.com/51927819/112325809-60a91000-8c8a-11eb-9218-1be7883e5019.png)
[Logs.zip](https://github.com/RainbowMiner/RainbowMiner/files/6197934/Logs.zip)

**Comment by RainbowMiner** [comment]
Yes, many old GPUs have no means to read the current power consumption. Could you please run `nvidia-smi` within a command window and post the result?

**Comment by VoltaVX** [comment]
![image](https://user-images.githubusercontent.com/51927819/112336602-99011c00-8c93-11eb-80fd-da50394fdcc7.png)

**Comment by RainbowMiner** [comment]
Just as suspected: this GPU doesn't report power usage nor load. But I'll add the GTX780 to my internal gdp database, then, at least, you will get a rough estimate.

**Comment by VoltaVX** [comment]
Alrighty, thank you so much!

**Comment by VoltaVX** [comment]
Good afternoon,

I updated to the latest version from master, but my card still shows "0W"

**Comment by RainbowMiner** [comment]
Could you please open `http://localhost:4000/devices` in your browser? It should now display a JSON encoded record - upload it here. Maybe I have misspelled the GPU's name.

**Comment by VoltaVX** [comment]
[
  {
    "Name": "GPU#01",
    "Index": 1,
    "PlatformId": 1,
    "PlatformId_Index": 0,
    "Type_PlatformId_Index": 0,
    "Platform_Vendor": "NVIDIA",
    "Vendor": "NVIDIA",
    "Vendor_Name": "NVIDIA Corporation",
    "Vendor_Index": 0,
    "Type_Vendor_Index": 0,
    "Type": "Gpu",
    "Type_Index": 1,
    "Type_Mineable_Index": 0,
    "BusId_Index": 1,
    "BusId_Type_Index": 1,
    "BusId_Type_Vendor_Index": 0,
    "BusId_Type_Mineable_Index": 0,
    "BusId_Vendor_Index": 0,
    "OpenCL": {
      "DeviceIndex": 0,
      "Name": "GeForce GTX 780",
      "Architecture": "Other",
      "Type": "Gpu",
      "Vendor": "NVIDIA Corporation",
      "GlobalMemSize": 3221225472,
      "GlobalMemSizeGB": 3,
      "MaxComputeUnits": 12,
      "PlatformVersion": "OpenCL 1.2 CUDA 11.2.135",
      "DriverVersion": "461.40",
      "PCIBusId": "01:00",
      "DeviceCapability": "3.5",
      "CardId": -1
    },
    "Model": "GTX780",
    "Model_Base": "GTX780",
    "Model_Name": "GeForce GTX 780",
    "InstanceId": "PCI\\VEN_10DE&DEV_1004&SUBSYS_17843842&REV_A1\\4&1B844C37&0&0008",
    "CardId": -1,
    "BusId": "01:00",
    "GpuGroup": "",
    "Data": {
      "AdapterId": 0,
      "Utilization": 100,
      "UtilizationMem": null,
      "Clock": null,
      "ClockMem": null,
      "FanSpeed": 100,
      "Temperature": 80,
      "PowerDraw": 0.0,
      "PowerLimit": null,
      "PowerLimitPercent": 0,
      "PowerMaxLimit": null,
      "PowerDefaultLimit": null,
      "Pstate": "P0",
      "Method": "smi"
    },
    "DataMax": {
      "Clock": 0,
      "ClockMem": 0,
      "Temperature": 80.0,
      "FanSpeed": 100,
      "PowerDraw": 0.0
    }
  },
  {
    "Name": "CPU#00",
    "Index": 2,
    "Vendor": "INTEL",
    "Vendor_Name": "GenuineIntel",
    "Type_PlatformId_Index": 0,
    "Type_Vendor_Index": 0,
    "Type": "Cpu",
    "Type_Index": 0,
    "Type_Mineable_Index": 0,
    "Model": "CPU",
    "Model_Base": "CPU",
    "Model_Name": "Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz",
    "Features": [
      "avx",
      "avx2",
      "bmi2",
      "fma3",
      "sse",
      "abm",
      "aes",
      "ssse3",
      "mpx",
      "sse42",
      "bmi1",
      "sse3",
      "sse2",
      "sse41",
      "rdrand",
      "x64",
      "mmx",
      "adx"
    ],
    "Data": {
      "Cores": 8,
      "Threads": 8,
      "CacheL3": 12288,
      "Clock": 4600,
      "Utilization": 88,
      "PowerDraw": 116,
      "Temperature": 83,
      "Method": "ohm"
    },
    "DataMax": {
      "Clock": 4600,
      "Utilization": 95,
      "PowerDraw": 120,
      "Temperature": 83
    }
  }
]
